k8sfv/run-test

```diff
@@ -58,6 +58,9 @@
 #
 # Typha logging level.
 : ${TYPHA_LOG_LEVEL:=info}
+#
+# CRD Manifest to use
+: ${CRD_URL:=https://raw.githubusercontent.com/projectcalico/libcalico-go/master/test/crds.yaml}
```

Comment:
This doesn't look right to me.  I don't think we should introduce a dependency on libcalico-go master for the FV tests when we have libcalico-go pinned in our vendor directory.  Why not get this from the file system at `./vendor/github.com/projectcalico/libcalico-go/test/crds.yaml`?